localization/english/wc_physical_lifestyle_l_english.yml

```diff
@@ -0,0 +1,21 @@
+﻿l_english:
+ physical_lifestyle_name:0 "Physical"
+ physical_lifestyle_desc:0 "Focus on training your body and mind, on acquiring the strength to defeat your enemies."
+
+ stamina_name:0 "Resilient"
```

Comment:
I'd personally change this to Resilience since you got an adjective here, but the strength one has a noun.